snippets/ts-snippets.json

```
@@ -222,5 +222,11 @@
       ""
     ],
     "description": "Creates a React component class with PropTypes with connected redux and ES7 module system and TypeScript interfaces"
+  },
+  "Export interface": {
+    "prefix": "expoint",

```

```suggestion
    "prefix": "expint",

```

I would like to not name anything as `expo`, because maybe in future some expo snippets will appear. :)